docs/en/tutorials/getting-started.rst

```diff
@@ -934,13 +927,13 @@ The last missing definition is that of the User entity:
 
             /**
              * @OneToMany(targetEntity="Bug", mappedBy="reporter")
-             * @var Bug[]
+             * @var Bug[] An ArrayCollection of Bug objects.
```

Comment:
Is it even acceptable to use `@var Bug[]` here? $reportedBugs is an object of type ArrayCollection.